packages/destination-actions/src/destinations/launchdarkly-audiences/index.ts

```diff
@@ -0,0 +1,68 @@
+import type { DestinationDefinition } from '@segment/actions-core'
+import type { Settings } from './generated-types'
+
+import syncAudience from './syncAudience'
+import { defaultValues } from '@segment/actions-core'
+import { CONSTANTS } from './constants'
+
+const destination: DestinationDefinition<Settings> = {
+  name: 'Launchdarkly Audiences',
+  slug: 'actions-launchdarkly-audiences',
+  mode: 'cloud',
+  description: 'Sync Segments audiences to LaunchDarkly Big Segments.',
+
+  authentication: {
+    scheme: 'custom',
+    fields: {
+      apiKey: {
+        label: 'API Key provided by the LaunchDarkly integration',
+        description:
+          'API Key used for [LaunchDarkly API authorization](https://app.launchdarkly.com/settings/authorization).',
+        type: 'password',
+        required: true
+      },
+      clientId: {
+        label: 'LaunchDarkly client-side ID',
+        description:
+          'Find and copy the [client-side ID](https://app.launchdarkly.com/settings/projects) in the LaunchDarkly account settings page.',
```

Comment:
My understanding is that we want to separate audiences by environments so we wouldnt want dev audiences to mix with prod audiences so I'm thinking it kinda goes with authentication.

Also, can you expand on what you mean by dynamic field? like fetching the all environments (requires a proj key) for the accounts and displaying it as a dropdown option?

Reply:
Segment has something called [dynamic fields](https://github.com/segmentio/action-destinations#dynamic-fields) that seems to be similar to our dynamic enum. I haven't tried it yet so I'm not sure if it will work for us (especially if we need to select project first), but I think it's worth trying.

Also the more I think about it, the more I think we should leave the the envID here.

Reply:
Let me test out the dynamic fields to see if we can use it.

Reply:
After some discussion and discovery we decided not to use dynamic fields for now. However, we will discuss it with the partner engineer.